### DISCUSSION TITLE
Inline Permissions

### DISCUSSION THREAD

**Discussion by GHAFRI** [requarks/wiki#6622] [Help / Questions] [answered]
Is there a way to have inline permissions such as a page that can be viewed by lets say and Admin, mod and support but have inline boxes that have specific permissions to see only, like certain text that can only be seen by an admin while support can still access the page at the same time. This will be very convenient instead of making the same content for each on different pages

**Comment by NGPixel** [comment, marked as answer]
No

**Reply by Ghafrigm** [reply]
Why not?

**Reply by NGPixel** [reply]
@Ghafrigm You asked whether it was possible. The answer is no...
